src/theme/base/index.css

```diff
@@ -182,8 +182,8 @@ select {
 3. Comment out because it relies too much on new layer() browser feature
 */
 
+/* [type='button'], */
```

Comment:
Should just remove if you want to remove it?

Reply:
Well, in an ideal situation we would un-comment it and re-implement it with the usage of CSS `@layers`, since that's what it's there for. But, if we end up dropping support for layers due to Safari versions, then I'm down to remove it.

Reply:
Gotcha okay, we can keep